### ISSUE TITLE
CupertinoSwitch does not respect the 'On/Off Labels' iOS accessibility system setting

### ISSUE THREAD

**Issue by omar-f** [flutter/flutter#65840]
## Steps to Reproduce
1. Run `flutter create bug`.
2. Update `main.dart` as follows:

```dart
import 'package:flutter/cupertino.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      home: CupertinoPageScaffold(
        child: SafeArea(
          child: SwitchDemo()
        ),
      ),
    );
  }
}


class SwitchDemo extends StatefulWidget {
  @override
  _SwitchDemoState createState() => _SwitchDemoState();
}

class _SwitchDemoState extends State<SwitchDemo> {

  var _isOn = true;

  @override
  Widget build(BuildContext context) {
    return CupertinoSwitch(
      value: _isOn,
      onChanged: (newValue) {
        setState(() {
          _isOn = newValue;
        });
      },
    );
  }
}
```

3. Steps to reproduce
- Run the app on an iOS device or simulator
- Open the device settings
- Go to Accessibility -> Display & Text Size. Turn on 'On/Off labels'. (this is the settings path as of iOS 13)
- Return to the Flutter app

**Expected results:**
The switch should show on/off labels, like this:

![off-labelled](https://user-images.githubusercontent.com/22119074/93197998-496df080-f744-11ea-9834-346559aeee24.png)
![on-labelled](https://user-images.githubusercontent.com/22119074/93198001-4a068700-f744-11ea-9ce8-f3958e7c279e.png)

**Actual results:** 
The switch is not labelled:
![off-notlabelled](https://user-images.githubusercontent.com/22119074/93198072-64d8fb80-f744-11ea-8098-5cb9b5f88e73.png)
![on-notlabelled](https://user-images.githubusercontent.com/22119074/93198073-64d8fb80-f744-11ea-944d-6d35db43c09b.png)

<details>
  <summary>Logs</summary>

`flutter doctor -v`

```
[✓] Flutter (Channel stable, 1.20.3, on Mac OS X 10.15.6 19G2021, locale en-GB)
    • Flutter version 1.20.3 at /Users/omarf/Developer/flutter
    • Framework revision 216dee60c0 (2 weeks ago), 2020-09-01 12:24:47 -0700
    • Engine revision d1bc06f032
    • Dart version 2.9.2


[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at /Users/omarf/Library/Android/sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.7)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.7, Build version 11E801a
    • CocoaPods version 1.8.4

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[!] IntelliJ IDEA Ultimate Edition (version 2020.1.2)
    • IntelliJ at /Users/omarf/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[!] VS Code (version 1.48.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (1 available)
    • iPhone 11 (mobile) • 96EEC63D-B411-4473-895B-2E2D7273DD9F • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)

! Doctor found issues in 2 categories
```

</details>

**Comment by darshankawar** [comment]
Issue replicable on latest master. Tested on iphone 6s.

<details>
<summary>flutter doctor -v</summary>

```
Flutter 1.22.0-10.0.pre.174 • channel master • https://github.com/flutter/flutter.git
Framework • revision 37b03ec2c2 (4 hours ago) • 2020-09-14 23:52:04 -0400
Engine • revision 7e962ba911
Tools • Dart 2.10.0 (build 2.10.0-124.0.dev)

Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 1.22.0-10.0.pre.174, on Mac OS X 10.15.4 19E2269, locale en-IN)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.46.1)
[✓] Connected device (4 available)

• No issues found!

```

</details>

**Comment by TheBiirb** [comment]
@darshankawar Looks like a duplicate of [#4830](https://github.com/flutter/flutter/issues/4830)

**Comment by darshankawar** [comment]
Thanks @TheBiirb for the tip.
Closing this as duplicate of [4830](https://github.com/flutter/flutter/issues/4830)
@omar-f please follow-up for updates and any questions in 4830.
Thanks.
